examples/angular/src/lib/shorten/BUILD.bazel

```diff
@@ -4,9 +4,8 @@ package(default_visibility = ["//:__subpackages__"])
 
 ts_library(
     name = "shorten",
-    srcs = ["shorten.ts"],
+    srcs = ["index.ts"],
```

Comment:
`module_root` not yet supported by linker

Reply:
yeah let's add a package.json here with a main field to fix this in a followup

Reply:
roger. I'll add a TODO